tests/e2e/karydia_default_network_policy_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+// This file is licensed under the Apache Software License, v. 2 except as
+// noted otherwise in the LICENSE file.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"os/exec"
+	"syscall"
+	"testing"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	Success int = 0
+	TimeOut int = 1
+)
+
+func execCommandAssertExitCode(t *testing.T, command string, expectedExitCode int) {
+	exitCode := Success
+
+	cmd := exec.Command("/bin/sh", "-c", command)
```

Comment:
I tried out many different ways but it seems to be the only way that works.
The problem with "exec.Command("kubectl", command)" is that the integration test never enters the path for successful exit code (number 0).
Perhaps the binary "kubectl" somehow works differently.

Reply:
It seems that you'll try to call the `Command` function not correctly.

Reply:
As the example shows at the above mentioned documentation you have to call it like `exec.Command("tr", "a-z", "A-Z")`.

Reply:
@SAPJonasZengerle @CodeClinch 
I have found no solution for it. I tried several possibilities, such as:
exec.Command("kubectl", command)
exec.Command("kubectl", "exec", command)
etc.
For some reason, the code never reaches the else section for successful test execution (exit code 0). 
In my opinion, it is okay, because it is a test and it is working.

Reply:
@SAPJonasZengerle 
Let's try to improve the Command() call after my holiday. Is it okay for you?

Reply:
FIXED.